src/Data/Functor/Linear.hs

```diff
@@ -28,7 +29,7 @@ class Functor f where
 -- drops values, which we are not allowed to do in a linear container).
 --
 -- In fact, an applicative functor is precisely a functor equipped with (pure
--- and) @zipWith :: (a ->. b ->. c) -> f a ->. f b ->. f c@
+-- and) @liftA2 :: (a ->. b ->. c) -> f a ->. f b ->. f c@
```

Comment:
Well, the reference to `zipWith` was rather on purpose. We probably want to say that `liftA2` has the same signature as `zipWith` does for lists or something.